javascript-operators-and-expressions/main.js

```diff
@@ -37,6 +37,6 @@ console.log('isSparta', isSparta);
 console.log('typeof isSparta:', typeof isSparta);
 
 var motto = fullName;
-motto = fullName + ' is the GOAT';
+motto = fullName += ' is the GOAT';
```

Comment:
This is now making changes to the `fullName` variable, which is not the intention. You should be adding onto the current value of `motto`